worker/src/RTC/Codecs/VP8.cpp

```diff
@@ -267,7 +267,10 @@ namespace RTC
 				if (this->payloadDescriptor->tlIndex > context->GetTargetTemporalLayer())
```

Comment:
Does anybody know why we check "GetTargetTemporalLayer()" here and in the next condition (5 lines below) we use "GetCurrentTemporalLayer()" instead?  Shouldn't both check for Current?

Reply:
Ufff to hard to remember now but of course that's on purpose. Can we rephrase the question? Why would it be wrong as it is?

Reply:
NVM, After rereading it I think the existing code is correct and I understand why those conditions use a different GetXXXXTemporalLayer().   Thx